includes/data-port/class-sensei-import-job-cli.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * File containing the Sensei_Import_Job_CLI class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * This class represents a data import job.
+ */
+class Sensei_Import_Job_CLI {
+	/**
+	 * Run the import command.
+	 *
+	 * @param array $args       Arguments passed without a name.
+	 * @param array $assoc_args Arguments passed with a name.
+	 */
+	public function __invoke( $args = [], $assoc_args = [] ) {
+		$files = [];
+		if ( ! empty( $assoc_args['questions'] ) ) {
```

Comment:
Maybe split the validation in a separate method?

Reply:
Do you think this class should go in production? I was thinking it would just be temporary, but could clean it up if you think not.

Reply:
As discussed in Slack, it would be great to add it doesn't need much work.

Reply:
b488b5c12fdb2f4707e8fce420d7b385ddc7d2c6